lyse/__init__.py

```diff
@@ -246,6 +246,7 @@ def save_result_array(self, name, data, group=None,
             if not group:
                 # Save dataset to results group by default
                 group = 'results/' + self.group
+                
```

Comment:
Could you remove this unnecessary change so that it doesn't clutter up the diff?